app/views/layouts/embed.html.erb

```diff
@@ -30,10 +31,10 @@ Unless required by applicable law or agreed to in writing, software distributed
     <%= stylesheet_link_tag "application", media: "all" %>
     <%= yield :page_styles %>
     <%= yield :additional_head_content %>
-    <%= render "modules/google_analytics" %>
   </head>
 
   <body>
+    <%= render_google_tag_manager_body %>
```

Comment:
Note: there wasn't an analytics script in the body element of `embed.html.erb` previously (though it did have the head script), so I'm not 100% sure adding this line is correct or necessary.